runner/local/simple_test.go

```diff
@@ -112,6 +114,10 @@ func aborted() runner.ProcessStatus {
 	return runner.AbortStatus(runner.RunId(""))
 }
 
+func badRequest(errorText string) runner.ProcessStatus {
+	return runner.BadRequestStatus(runner.RunId(""), fmt.Errorf(errorText))
+}
+
 func assertRun(t *testing.T, r runner.Runner, expected runner.ProcessStatus, args ...string) runner.RunId {
```

Comment:
I've found it useful for commonly used test assertion routines to have a 'tag' parameter that the test can set to better identify which use of the assertion is error-ing.